presidio-analyzer/app.py

```diff
@@ -62,7 +62,7 @@ def analyze() -> Tuple[str, int]:
                 return (
                     json.dumps(
                         recognizer_result_list,
-                        default=lambda o: o.to_json(),
+                        default=lambda o: o.__dict__,
```

Comment:
This is a potential point of failure. It would be better to create a `to_dict` method on `RecognizerResult` as some fields in the class are either not wanted in the response or are not handled correctly using `self.__dict__` (for example non-primitive types). 

Not urgent for now but would make the code a little more robust.

Reply:
Added a to_dict method, currently returning self.__dict__, for a preserving behaviour